talk/expert/cpp20spaceship.tex

```diff
@@ -114,7 +114,9 @@
   \end{exampleblock}
```

Comment:
Line https://github.com/hsf-training/cpluspluscourse/pull/217/files#diff-a6dec9ca6d8f983ed52847bf233bc288a103be99d342571e6717bf548e042837R105 above leaks out of the box, but I'm not allowed to comment there.

Reply:
Next commit will fix that.